### DISCUSSION TITLE
Unexpected behavior of the identify -format %A command

### DISCUSSION THREAD

**Discussion by matthias8283** [ImageMagick/ImageMagick#5108] [Help] [answered]
Hi,

I have the following case, that seems to be a bug:

Given:
* ImageMagick 7.1.0-31 Q16-HDRI
* A TIF file with an alpha channel

When:
* Executing the command
`magick identify -format "%A" test.tif`

Then:
* I expect the output "true" (see also https://imagemagick.org/script/escape.php ), but currently the output is "Bend".

Is this correct or is it a bug?

Best,
Matthias

**Comment by snibgo** [comment, marked as answer]
The documentation is out-of-date.

Values that may be returned are: "Undefined", "Blend", "Copy" and "Update". "Undefined" corresponds to the old "false", and the others to "true".

**Reply by urban-warrior** [reply]
We updated the "escapes" web page.   FYI

**Reply by matthias8283** [reply]
Thanks to both of you! Have a nice day!
